Composer/packages/client/src/components/CreationFlow/CreationFlow.tsx

```diff
@@ -129,6 +133,11 @@ const CreationFlow: React.FC<CreationFlowProps> = () => {
       navigate(`./QnASample/importQnA`);
       return;
     }
+    if (templateId === 'va-core') {
```

Comment:
now that this function is not only handling the QnA, please rename the function accordingly.

Reply:
Is there a particular reason that the QnA route and templateId values are campleCase but va-core is hyphen-case?

Reply:
Fixed function naming and route syntax